src/MPQEditor/MPQCircleSelector.ts

```diff
@@ -54,6 +54,63 @@ export class MPQSelectionPanel
 
   public static register(_context: vscode.ExtensionContext): void {}
 
+  public static createOrShow(
+    extensionUri: vscode.Uri,
+    args: MPQSelectionCmdOpenArgs,
+    handler: MPQSelectionEvent | undefined
+  ) {
+    let column = args.panel.viewColumn;
+    if (column) {
+      if (column >= vscode.ViewColumn.One) {
+        column = column + 1;
+      }
+    }
+
+    // search for existing panel
+    const oldPanel = MPQSelectionPanel.findSelPanel(
+      args.modelPath,
+      args.document.uri.toString()
+    );
+    if (oldPanel) {
+      oldPanel._panel.reveal(column);
+      return;
+    }
+
+    // Otherwise, create a new panel.
+    const lastSlash = args.modelPath.lastIndexOf(path.sep) + 1;
```

Comment:
👍  By using this, we can expect that it will work on Windows too. (Otherwise, seperator differs on OS so it won't work). But it wouldn't harm to check the functionality on Windows.